tool/firebase_to_json.py

```diff
@@ -99,9 +102,12 @@ def _usage_and_exit(error_message):
 
     data = {}
 
+    short_id = tool_utils.short_id()
+    log.audit(f"firebase_to_json: JobID {short_id}")
     for key in root_keys_to_export:
```

Comment:
Do we want to `log.notify` the keys that are exported?

Reply:
Good idea

Reply:
Done